specs/credentialmanagement/index.src.html

```diff
@@ -821,9 +867,10 @@ <h4 id="interfaces-request-options">
 
     <pre>
       navigator.<a idl>credentials</a>.<a idl lt="get()">get</a>({
-        "<a idl>types</a>": [ "federated" ],
-        "<a idl>options</a>": {
-            "<a idl>providers</a>": [ "https://identity.example.com/" ]
+        "<a idl>filters</a>": {
+            "federated" : {
+                "<a idl>providers</a>": [ "https://identity.example.com/" ]
```

Comment:
This adds another layer of indirection. We've moved from:

```
.get({
  'federations': [ ... ]
})
```

to

```
.get({
  'options': {
    'providers': [ ... ]
  }
})
```

to

```
.get({
  'filters': {
    'federated': {
      'providers': [ ... ]
    }
  }
})
```

Maybe it's reasonable to accept that complexity if it actually solves use cases that folks care about. I feel less sanguine about accepting it when folks who care about those use cases don't think it's going to be enough.